processing/data_collection/gazette/pipelines.py

```diff
@@ -22,6 +22,21 @@ def process_item(self, item, spider):
         return item
 
 
+class DefaultValuesPipeline:
+    """ Add defaults values field, if not already set in the item """
+
+    default_field_values = {
+        "territory_id": lambda item, spider: getattr(spider, "TERRITORY_ID"),
```

Comment:
**suggestion**: If a spider doesn't have a `TERRITORY_ID` set, it will raise an `AttributeError`. You could add a default value if this happens: `getattr(spider, "TERRITORY_ID", "UNKNOWN")`.

Reply:
I think this is intended, right?
Shouldn't all spiders should define a `TERRITORY_ID`? And in case it is missing, it should fail.
Maybe, this is not the right place to raise this error, but if the field is required, it should be checked somewhere to avoid yielding items without this info

Reply:
Yeah, good points. Both of you :)
Let me add a check for that at `base.py`